src/LibraryManager.Vsix/Commands/ManageLibrariesCommand.cs

```diff
@@ -44,20 +44,24 @@ private void BeforeQueryStatus(object sender, EventArgs e)
         private void Execute(object sender, EventArgs e)
         {
             Telemetry.TrackUserTask("ManageLibraries");
-            Project project = VsHelpers.DTE.SelectedItems.Item(1).Project;
-            string rootFolder = project.GetRootFolder();
 
-            string configFilePath = Path.Combine(rootFolder, Constants.ConfigFileName);
+            Project project = VsHelpers.GetProjectOfSelectedItem();
 
-            if (File.Exists(configFilePath))
+            if (project != null)
             {
-                VsHelpers.DTE.ItemOperations.OpenFile(configFilePath);
-            }
-            else
-            {
-                System.Threading.Tasks.Task.Run(async () =>
+                string rootFolder = project.GetRootFolder();
+
+                string configFilePath = Path.Combine(rootFolder, Constants.ConfigFileName);
+
+                if (File.Exists(configFilePath))
                 {
-                    CancellationToken token = CancellationToken.None;
+                    VsHelpers.DTE.ItemOperations.OpenFile(configFilePath);
+                }
+                else
+                {
+                    System.Threading.Tasks.Task.Run(async () =>
+                    {
+                        CancellationToken token = CancellationToken.None;
 
                     if (!File.Exists(configFilePath))
```

Comment:
Fix this indenting. Must've been a merge conflict with the new System.Threading.Tasks.Task.Run block that was added.